src/resources/views/crud/fields/relationship.blade.php

```diff
@@ -20,6 +20,7 @@
     // if field is not ajax but user wants to use InlineCreate
     // we make minimum_input_length = 0 so when user open we show the entries like a regular select
     $field['minimum_input_length'] = ($field['ajax'] !== true) ? 0 : ($field['minimum_input_length'] ?? 2);
+    $field['subfields'] = $field['subfields'] ?? $field['fields'] ?? $field['pivotFields'] ?? false;
```

Comment:
We can't "accept" any of those so downstream and only account for `subfields` in the crud create/update upstream process. 

It works in terms of field definition and interface, but if you do:

```php
CRUD::field('dummyproducts')->pivotFields([
            [
                'name' => 'notes'
            ], 
        ]);
```

It will works as interface as I said, but since the "core" backpack does not know about `pivotFields` only `subfields`, the field does not work as expected when updating. @tabacitu you can test this branch with any `BelongsToMany` with pivot where you define `pivotFields` instead of `subfields`. Notice that the field will work, and the create process will work too (because it gets the values from the request and not from the fields) .. but when displaying (on update), the value you saved in database relation will not display. That is because upstream we get values for `subfields` fields, not `pivotFields` fields.

- When using with `pivotFields`:
![image](https://user-images.githubusercontent.com/7188159/149369047-c5ff6304-6174-4c05-b740-6e240c9cd4c7.png)

-When I change `pivotFields` to `subfields` the information is displayed correctly:
![image](https://user-images.githubusercontent.com/7188159/149369194-ef2da91c-bc16-41a8-9f55-bbea5f4a6342.png)

Question: ditch `pivotFields`, `fields` in favor of only `subfields` OR account for the previouos is core functions ?

I vouch for ditch! Last word is on you. @tabacitu

Reply:
```suggestion
    $field['subfields'] = $field['subfields'] ?? false;
```

Reply:
Good  catch! Ok then, `subfields` everywhere (with `fields` also working in `repeatable` for backwards-compatibility).